src/Microsoft.Diagnostics.Monitoring.RestServer/Controllers/DiagController.cs

```diff
@@ -364,12 +364,22 @@ public Task<ActionResult<IEnumerable<Models.ProcessIdentifier>>> GetProcesses()
 
                     loggerFactory.AddProvider(new StreamingLoggerProvider(outputStream, format, level));
 
-                    var settings = new EventLogsPipelineSettings
+                    var settings = new EventLogsPipelineSettings()
                     {
-                        Duration = duration,
-                        LogLevel = level,
+                        Duration = duration
                     };
 
+                    //If the user does not explicitly specify a log level, we will use the apps defaults for .net 5.
+                    //Otherwise, we use the log level specified by the query.
+                    if ((!level.HasValue) && (processInfo.EndpointInfo.RuntimeInstanceCookie != Guid.Empty))
```

Comment:
Might have to update this heuristic when we generate artificial cookie values for .NET Core 3.1, but we can probably also add another field to specify whether the cookie is real or generated in order to differentiate their usage.

Reply:
I was thinking we would another field to indicated generated runtime id's.